internal/apis/admin/v1/types.go

```diff
@@ -125,11 +125,13 @@ type Upstream struct {
 
 	Name string `json:"name"`
 
-	HashOn       string        `json:"hash_on,omitempty"`
-	HashOnHeader string        `json:"hash_on_header,omitempty"`
-	HashFallback string        `json:"hash_fallback,omitempty"`
-	Healthchecks *Healthchecks `json:"healthchecks,omitempty"`
-	Slots        int           `json:"slots,omitempty"`
+	HashOn           string        `json:"hash_on,omitempty"`
+	HashFallback     string        `json:"hash_fallback,omitempty"`
+	HashOnHeader     string        `json:"hash_on_header,omitempty"`
+	HashOnCookie     string        `json:"hash_on_cookie,omitempty"`
+	HashOnCookiePath string        `json:"hash_on_cookie_path,omitempty"`
```

Comment:
Reading the [documentation](https://docs.konghq.com/0.14.x/admin-api/#add-upstream), it seems like we are also missing `hash_fallback_header` property.
Could you take care of it in this PR?

Reply:
I added the hash fallback header config setting propagation as well